api/assets/constants/constants.js

```diff
@@ -53,8 +53,8 @@ module.exports = {
     text: fs.readFileSync(path.join(__dirname, 'RSVP_Email.html'), 'utf-8'),
   },
   GCS: {
-    resumeBucket: process.env.APP_ENV === 'test' ? 'hackpsu-resumes-test' : 'hackpsuf2018-resumes',
-    travelReimbursementBucket: process.env.APP_ENV === 'test' ? 'hackpsu2018-travel-reimbursement-receipts-test' : 'hackpsuf2018-travel-reimbursement-receipts',
+    resumeBucket: process.env.APP_ENV === 'test' ? 'hackpsu-resumes-test' : 'hackpsus2019-resumes',
+    travelReimbursementBucket: process.env.APP_ENV === 'test' ? 'hackpsu2018-travel-reimbursement-receipts-test' : 'hackpsus2019-travel-reimbursement-receipts',
```

Comment:
'hackpsu2018-travel-reimbursement-receipts-test' should be changed to 'hackpsu2019-travel-reimbursement-receipts-test' ?

Reply:
touche... good catch